OpenRA.Test/OpenRA.Test.csproj

```diff
@@ -19,7 +19,17 @@
     <DebugType>full</DebugType>
     <PlatformTarget>x86</PlatformTarget>
     <ErrorReport>prompt</ErrorReport>
-    <CodeAnalysisRuleSet>ManagedMinimumRules.ruleset</CodeAnalysisRuleSet>
+    <CodeAnalysisRuleSet>AllRules.ruleset</CodeAnalysisRuleSet>
```

Comment:
This affects the ruleset used when VS (and other tools?) do static code analysis.

I am changing only to make it consistent with all the others.